app/views/home/maintainers.html.erb

```diff
@@ -1,27 +1,27 @@
 <div class="bg-gray mb-6 border-bottom jumbotron px-3">
   <div class="container-md">
-    <h2 class="alt-h1 text-center text-red">Help others pitch in</h2>
+    <h2 class="alt-h1 text-center text-red">Building software is more fun with others.</h2>
```

Comment:
> I think we should still explicitly appeal to a maintainer's personal need (to not have to work so hard on their project).

I agree. Maybe `Build software with help from others`? That's long I realise but I think it nudges towards the idea that people will help you. Not sure about "fun".